packages/bpk-component-calendar/src/BpkCalendarGrid.js

```diff
@@ -74,10 +70,6 @@ class BpkCalendarGrid extends Component {
           nextProps.month,
           nextProps.weekStartsOn,
         ),
-        daysOfWeek: orderDaysOfWeek(
```

Comment:
This has been removed but is a key part of the calendar for providing information on the calendar (see prop description) for different styles/localised info on the week days are we confident in removing it won't impact consumers:

https://github.com/Skyscanner/backpack/tree/main/packages/bpk-component-calendar#daysofweek

Reply:
Yes, I believe so. In the grid, it is only used by the components that were removed, so without them it is flagged as unused variable. I double checked the code and indeed it is set in the constructor, but never really used.